owncloudApp/src/main/res/layout/item_file_list.xml

```diff
@@ -95,6 +95,37 @@
             app:layout_constraintTop_toBottomOf="@id/Filename"
             tools:text="Mod Date" />
 
+        <ImageView
```

Comment:
This one could be extracted to a new component and used wherever we need it. At the moment, we have the same item at two different places and we will need to update both to keep consistency.

A new component could keep the icon, space name, and parent path. Just an idea..

Reply:
I extracted this to a new component, `space_path_line.xml` 👍